src/experiments/super_tiles/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import useTreatment from '../../hooks/useTreatment';
+import IOBanner from '../../components/IOBanner';
+import CodeStreamTile from './components/CodeStreamTile';
+import GuidedInstallTile from './components/GuidedInstallTile';
+
+const SuperTilesExperiment = ({ isMobile }) => {
+  const { treatment } = useTreatment('super_tiles');
+
+  if (treatment === 'on') {
+    return (
+      <div
+        css={css`
+          display: flex;
+          flex-direction: row;
+          justify-content: space-between;
+          margin-bottom: 16px;
+          gap: 16px;
```

Comment:
We should probably consider stacking these tiles on mobile / smaller screen widths. It looks a little wonky on mobile at the moment (very squished).

Reply:
Sounds good - I converted this to a grid instead.